src/Compilers/CSharp/Portable/Symbols/TypeParameterBounds.cs

```diff
@@ -72,4 +85,20 @@ private TypeParameterBounds()
         /// </summary>
         public readonly TypeSymbol DeducedBaseType;
     }
+
+    internal static class TypeParameterBoundsExtensions
+    {
+        internal static bool IsSet(this TypeParameterBounds boundsOpt, bool early)
```

Comment:
Why make this an extension, rather than an instance method?

Reply:
To allow a `null` bounds parameter which is a valid `TypeParameterBounds` value.

---
In reply to: [195545011](https://github.com/dotnet/roslyn/pull/27525#discussion_r195545011) [](ancestors = 195545011)